src/plugins/intel_cpu/src/cpu_types.cpp

```diff
@@ -187,7 +187,11 @@ static const TypeToNameMap& get_type_to_name_tbl() {
         {"Ceiling", Type::Math},
         {"Cos", Type::Math},
         {"Cosh", Type::Math},
+#if defined(OPENVINO_ARCH_ARM)
         {"Floor", Type::Math},
+#else
+        {"Floor", Type::Eltwise},
+#endif
```

Comment:
Why we need any arch specific conditions there? I would expect Floor to have Eltwise type for any platform

Reply:
`Floor` for ARM32 is part of `Math` operation, as before.

Reply:
done, thanks